dev/NavigationView/NavigationView.xaml

```diff
@@ -452,12 +443,15 @@
                                             VerticalContentAlignment="Stretch"
                                             HorizontalContentAlignment="Stretch"
                                             Grid.Row="7" />
-                                        <Grid
+                                        <!-- Left footer menu ItemsRepeater -->
+                                        <local:ItemsRepeater
```

Comment:
what should the automation  index/size be, is it separate for the footer items or is it part of the main items ?

Reply:
Right now, the main items are one list and the Settings item is another list with index "1 of 1". I think the right behavior would be to make the bottom items into a list that includes the Settings item. We also want to make sure the keyboard behavior matches this.

So  given a list:
- main1
- main2

- bottom1
- bottom2
- settings

Users should be able to Tab between these two lists. 
When keyboard focus is on main2, Narrator should announce it as "2 of 2"; pressing Up arrow moves focus to main1; pressing Down arrow moves focus to bottom1; pressing Tab moves focus to bottom1 (unless an item in the bottom list is selected, in which case focus should go to that item).
When keyboard focus is on bottom1, Narrtor should announce it as "1 of 3"; pressing Up arrow moves focus to main2; pressing Down arrow moves focus to bottom2; pressing Tab moves focus to the first focusable item in NavigationView's Header or Content (same behavior as on Settings item today).
When keyboard focus is on settings, Narrator should announce it as "3 of 3"; pressing Up arrow moves focus to bottom2; pressing Down arrow does nothing (same as today); pressing Tab moves focus to a Header or Content item (same as today).

Reply:
>  I think the right behavior would be to make the bottom items into a list that includes the Settings item. 
that's just what i did

>(unless an item in the bottom list is selected, in which case focus should go to that item)
this is broken in PR

>Narrator should announce it as "3 of 3"
Strange on Bottom1 Narrator announce  "1 of 3", but on Settings "1 of 1". I'll check this.

Everything else working as you described.

Reply:
@YuliKl How should Home and End work when main2 is selected? When bottom2 is selected?

Reply:
Home and End move focus within a list. We've said that this work will result in two lists, so
- if focus is on main2 - Home should move focus to main1; End should do nothing
- is focus is on bottom2 - Home should move focus to bottom1; End should move focus to settings